tests/firewall/firewall-06-tls-sni-enforce/test.yaml

```diff
@@ -52,17 +39,10 @@ checks:
     match:
       event_type: drop
 - filter:
-    min-version: 8
```

Comment:
This is less precise than #2179 as discussed in https://github.com/OISF/suricata-verify/pull/2173

Reply:
Discussion was rather in https://github.com/OISF/suricata-verify/pull/2174